.github/workflows/matlab.yml

```diff
@@ -0,0 +1,93 @@
+name: MATLAB Tests Workflow
+
+on:
+  push:
+  pull_request:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Execute a "nightly" build at 2 AM UTC 
+  - cron:  '0 2 * * *'
+  
+jobs:
+  build-matlab-tests:    
+    name: '[matlab:${{ matrix.matlab_version }}:${{ matrix.os }}]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        build_type: [Release]
+        os: [ubuntu-20.04]
+        matlab_version: [R2020a, R2020b, R2021a, latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
+        channels: conda-forge,robotology
+
+    - name: Setup MATLAB
+      uses: matlab-actions/setup-matlab@v1
+      with:
+        release: ${{ matrix.matlab_version }}
+
+    # workaround for https://github.com/robotology/robotology-superbuild/issues/64
+    - name: Do not use MATLAB's stdc++ to avoid incompatibilities with other libraries
+      if: contains(matrix.os, 'ubuntu')
+      run:
+          echo 'LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6' >> $GITHUB_ENV
+
+    - name: Dependencies
+      shell: bash -l {0}
```

Comment:
To avoid specifying that each shell is login you can set a [default shell](https://github.com/conda-incubator/setup-miniconda#use-a-default-shell).

Reply:
Done in https://github.com/robotology/idyntree/pull/917/commits/695d8131b69aeeb41ddbe17f73ad4461dab49aab .